aspnetcore/mvc/advanced/custom-model-binding/sample/CustomModelBindingSample/Startup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
Ok, but if they run into trouble they have no exception pages or logging to help them...